[2024-12-17 17:09:19,615] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Session key updated (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2510)
[2024-12-17 17:09:19,744] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Session key updated (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2510)
[2024-12-17 17:42:48,858] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Group coordinator 172.30.2.207:9092 (id: 2147483647 rack: null) is unavailable or invalid due to cause: session timed out without receiving a heartbeat response. isDisconnected: false. Rediscovery will be attempted. (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1000)
[2024-12-17 17:50:09,090] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Requesting disconnect from last known coordinator 172.30.2.207:9092 (id: 2147483647 rack: null) (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1013)
[2024-12-17 17:43:07,139] INFO [Consumer clientId=mysql-sink-cluster-configs, groupId=mysql-sink-cluster] Disconnecting from node 0 due to request timeout. (org.apache.kafka.clients.NetworkClient:871)
[2024-12-17 17:43:07,324] INFO [Consumer clientId=mysql-sink-cluster-offsets, groupId=mysql-sink-cluster] Disconnecting from node 0 due to request timeout. (org.apache.kafka.clients.NetworkClient:871)
[2024-12-17 17:50:09,113] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Client requested disconnect from node 2147483647 (org.apache.kafka.clients.NetworkClient:344)
[2024-12-17 17:50:09,137] INFO [Producer clientId=mysql-sink-cluster-configs] Node 0 disconnected. (org.apache.kafka.clients.NetworkClient:1021)
[2024-12-17 17:50:09,138] INFO [Producer clientId=mysql-sink-cluster-configs] Cancelled in-flight METADATA request with correlation id 488 due to node 0 being disconnected (elapsed time since creation: 53ms, elapsed time since send: 53ms, throttle time: 0ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:09,140] INFO [Consumer clientId=mysql-sink-cluster-configs, groupId=mysql-sink-cluster] Cancelled in-flight FETCH request with correlation id 108392 due to node 0 being disconnected (elapsed time since creation: 30022ms, elapsed time since send: 30022ms, throttle time: 0ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:09,143] INFO [Consumer clientId=mysql-sink-cluster-configs, groupId=mysql-sink-cluster] Error sending fetch request (sessionId=1575572714, epoch=108023) to node 0: (org.apache.kafka.clients.FetchSessionHandler:618)
org.apache.kafka.common.errors.DisconnectException
[2024-12-17 17:50:09,145] INFO [Consumer clientId=mysql-sink-cluster-offsets, groupId=mysql-sink-cluster] Cancelled in-flight FETCH request with correlation id 108191 due to node 0 being disconnected (elapsed time since creation: 30029ms, elapsed time since send: 30029ms, throttle time: 0ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:09,157] INFO [Consumer clientId=mysql-sink-cluster-offsets, groupId=mysql-sink-cluster] Error sending fetch request (sessionId=826404844, epoch=107822) to node 0: (org.apache.kafka.clients.FetchSessionHandler:618)
org.apache.kafka.common.errors.DisconnectException
[2024-12-17 17:50:09,145] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Cancelled in-flight HEARTBEAT request with correlation id 18471 due to node 2147483647 being disconnected (elapsed time since creation: 449277ms, elapsed time since send: 449277ms, throttle time: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:09,180] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Cancelled in-flight HEARTBEAT request with correlation id 18472 due to node 2147483647 being disconnected (elapsed time since creation: 446273ms, elapsed time since send: 446273ms, throttle time: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:09,181] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Cancelled in-flight HEARTBEAT request with correlation id 18473 due to node 2147483647 being disconnected (elapsed time since creation: 443270ms, elapsed time since send: 443270ms, throttle time: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:12,113] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Broker coordinator was unreachable for 3000ms. Revoking previous assignment Assignment{error=0, leader='connect-172.30.2.207:8083-32a35daa-4ae8-4e3a-846b-1d294614d0ac', leaderUrl='http://172.30.2.207:8083/', offset=1069, connectorIds=[], taskIds=[], revokedConnectorIds=[], revokedTaskIds=[], delay=0} to avoid running tasks while not being a member the group (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:147)
[2024-12-17 17:50:12,113] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Finished stopping tasks in preparation for rebalance (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2735)
[2024-12-17 17:50:12,114] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Finished flushing status backing store in preparation for rebalance (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2756)
[2024-12-17 17:50:12,149] INFO [Producer clientId=mysql-sink-cluster-statuses] Node 1 disconnected. (org.apache.kafka.clients.NetworkClient:1021)
[2024-12-17 17:50:12,150] WARN [Producer clientId=mysql-sink-cluster-statuses] Connection to node 1 (/172.30.2.147:9092) could not be established. Node may not be available. (org.apache.kafka.clients.NetworkClient:849)
[2024-12-17 17:50:12,151] INFO [AdminClient clientId=mysql-sink-cluster-shared-admin] Node 1 disconnected. (org.apache.kafka.clients.NetworkClient:1021)
[2024-12-17 17:50:12,151] WARN [AdminClient clientId=mysql-sink-cluster-shared-admin] Connection to node 1 (/172.30.2.147:9092) could not be established. Node may not be available. (org.apache.kafka.clients.NetworkClient:849)
[2024-12-17 17:50:12,152] INFO [Consumer clientId=mysql-sink-cluster-offsets, groupId=mysql-sink-cluster] Node 1 disconnected. (org.apache.kafka.clients.NetworkClient:1021)
[2024-12-17 17:50:12,152] WARN [Consumer clientId=mysql-sink-cluster-offsets, groupId=mysql-sink-cluster] Connection to node 1 (/172.30.2.147:9092) could not be established. Node may not be available. (org.apache.kafka.clients.NetworkClient:849)
[2024-12-17 17:50:12,153] INFO [AdminClient clientId=mysql-sink-cluster-shared-admin] Metadata update failed (org.apache.kafka.clients.admin.internals.AdminMetadataManager:273)
org.apache.kafka.common.errors.TimeoutException: Timed out waiting for a node assignment. Call: fetchMetadata
[2024-12-17 17:50:12,151] INFO [Producer clientId=mysql-sink-cluster-configs] Node 1 disconnected. (org.apache.kafka.clients.NetworkClient:1021)
[2024-12-17 17:50:12,153] WARN [Producer clientId=mysql-sink-cluster-configs] Connection to node 1 (/172.30.2.147:9092) could not be established. Node may not be available. (org.apache.kafka.clients.NetworkClient:849)
[2024-12-17 17:50:39,217] INFO [Producer clientId=mysql-sink-cluster-offsets] Disconnecting from node 1 due to request timeout. (org.apache.kafka.clients.NetworkClient:871)
[2024-12-17 17:50:39,217] INFO [Producer clientId=mysql-sink-cluster-offsets] Cancelled in-flight METADATA request with correlation id 367 due to node 1 being disconnected (elapsed time since creation: 30118ms, elapsed time since send: 30118ms, throttle time: 0ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:39,307] INFO [Consumer clientId=mysql-sink-cluster-configs, groupId=mysql-sink-cluster] Disconnecting from node 1 due to request timeout. (org.apache.kafka.clients.NetworkClient:871)
[2024-12-17 17:50:39,308] INFO [Consumer clientId=mysql-sink-cluster-configs, groupId=mysql-sink-cluster] Cancelled in-flight METADATA request with correlation id 108393 due to node 1 being disconnected (elapsed time since creation: 30128ms, elapsed time since send: 30128ms, throttle time: 0ms, request timeout: 30000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:52,145] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Disconnecting from node 1 due to request timeout. (org.apache.kafka.clients.NetworkClient:871)
[2024-12-17 17:50:52,146] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Cancelled in-flight METADATA request with correlation id 18475 due to node 1 being disconnected (elapsed time since creation: 42963ms, elapsed time since send: 42963ms, throttle time: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:52,148] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Cancelled in-flight FIND_COORDINATOR request with correlation id 18474 due to node 1 being disconnected (elapsed time since creation: 43033ms, elapsed time since send: 42962ms, throttle time: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:52,148] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Cancelled in-flight FIND_COORDINATOR request with correlation id 18476 due to node 1 being disconnected (elapsed time since creation: 40029ms, elapsed time since send: 40029ms, throttle time: 0ms, request timeout: 40000ms) (org.apache.kafka.clients.NetworkClient:364)
[2024-12-17 17:50:52,277] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Discovered group coordinator 172.30.2.207:9092 (id: 2147483647 rack: null) (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:937)
[2024-12-17 17:50:52,278] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Rebalance started (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:243)
[2024-12-17 17:50:52,278] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] (Re-)joining group (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:605)
[2024-12-17 17:50:52,286] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] JoinGroup failed: The coordinator is not aware of this member. Need to re-join the group. Sent generation was Generation{generationId=666, memberId='connect-172.30.2.207:8083-32a35daa-4ae8-4e3a-846b-1d294614d0ac', protocol='sessioned'} (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:683)
[2024-12-17 17:50:52,287] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Resetting generation and member id due to: encountered UNKNOWN_MEMBER_ID from JOIN_GROUP response (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1056)
[2024-12-17 17:50:52,287] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Request joining group due to: rebalance failed due to 'The coordinator is not aware of this member.' (UnknownMemberIdException) (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:1103)
[2024-12-17 17:50:52,287] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Rebalance started (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:243)
[2024-12-17 17:50:52,287] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] (Re-)joining group (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:605)
[2024-12-17 17:50:52,295] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] (Re-)joining group (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:605)
[2024-12-17 17:50:52,301] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Successfully joined group with generation Generation{generationId=668, memberId='connect-172.30.2.207:8083-b305fee0-76ad-4288-a0ba-672733da6973', protocol='sessioned'} (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:666)
[2024-12-17 17:50:52,310] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Successfully synced group in generation Generation{generationId=668, memberId='connect-172.30.2.207:8083-b305fee0-76ad-4288-a0ba-672733da6973', protocol='sessioned'} (org.apache.kafka.connect.runtime.distributed.WorkerCoordinator:843)
[2024-12-17 17:50:52,311] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Joined group at generation 668 with protocol version 2 and got assignment: Assignment{error=0, leader='connect-172.30.2.207:8083-b305fee0-76ad-4288-a0ba-672733da6973', leaderUrl='http://172.30.2.207:8083/', offset=1071, connectorIds=[], taskIds=[], revokedConnectorIds=[], revokedTaskIds=[], delay=0} with rebalance delay: 0 (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2648)
[2024-12-17 17:50:52,312] WARN [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Catching up to assignment's config offset. (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1811)
[2024-12-17 17:50:52,312] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Current config state offset 1069 is behind group assignment 1071, reading to end of config log (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1884)
[2024-12-17 17:50:52,323] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Finished reading to end of log and updated config snapshot, new config log offset: 1071 (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1911)
[2024-12-17 17:50:52,323] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Starting connectors and tasks using config offset 1071 (org.apache.kafka.connect.runtime.distributed.DistributedHerder:1979)
[2024-12-17 17:50:52,324] INFO [Worker clientId=connect-172.30.2.207:8083, groupId=mysql-sink-cluster] Finished starting connectors and tasks (org.apache.kafka.connect.runtime.distributed.DistributedHerder:2008)
[2024-12-17 17:55:12,279] INFO [Consumer clientId=mysql-sink-cluster-configs, groupId=mysql-sink-cluster] Node 1 disconnected. (org.apache.kafka.clients.NetworkClient:1021)
